Dockerfile

```diff
@@ -0,0 +1,53 @@
+# ---------------------------------------------------------------------------- #
+#                               Dymension builder                              #
+# ---------------------------------------------------------------------------- #
+
+FROM golang:1.18-bullseye as dymension-builder
+
+RUN apt-get update -y && apt-get install -y build-essential
+
+ENV PACKAGES curl make git bash gcc python3 wget
+
+RUN apt-get install -y $PACKAGES
+
+WORKDIR /app
+
+RUN git clone https://github.com/dymensionxyz/dymension && cd dymension && make install
+
+# ---------------------------------------------------------------------------- #
+#                                    Rollapp                                   #
+# ---------------------------------------------------------------------------- #
+
+FROM golang:1.18-bullseye
+COPY --from=dymension-builder /go/bin/dymd /go/bin
+
+RUN apt-get update -y && apt-get install -y build-essential
+
+ENV PACKAGES curl make git bash gcc python3 wget
+
+RUN apt-get install -y $PACKAGES
+
+WORKDIR /app
+
+COPY go.mod go.sum* ./
+
+RUN go mod download
+
+COPY . .
+
+RUN go install ./cmd/rollappd
+
+# Init rollapp script env vars
```

Comment:
why u need to set those in the dockerfile?

Reply:
it's redundant. Used it before but forgot to remote it. will remove it.